dal-identification.yml

```diff
@@ -2955,23 +2955,23 @@ components:
         telephone:
           type: string
           maxLength: 10
-          pattern: '^\d{10}$'
+          #pattern: '^\d{10}$'
         extension:
           type: string
           maxLength: 7
-          pattern: '^\d{0,7}$'
+          #pattern: '^\d{0,7}$'
```

Comment:
an extension number should be only numbers and no string. why did we deactivate this validation rule?

Reply:
@franciskonchou rappel-tu pourquoi tu as faite cette changement?